### PR TITLE
add PRs to Core triage project automation

### LINKED ISSUE #31: add PRs to automation for core triage project
the code looks like it's adding PRs, but doesn't seem to be. confirmed with this one: https://github.com/dbt-labs/dbt-bigquery/pull/284

which is still not showing up here: https://github.com/orgs/dbt-labs/projects/22/views/20

I vaguely recall PRs not working for some reason

### PR DESCRIPTION
**Pull request by lostmygithubaccount**
closes #31 

some notable things:

- see linked issue, can't `filterBy` PRs for some reason
- hit a 503 while testing, should add retry logic and whatnot 
- simply adds last 100 PRs for each repo on a schedule, should be fine

### CODE REVIEW

File: scripts/core-triage/graphql_queries.py
Comment (leahwicz) [inline scripts/core-triage/graphql_queries.py:48]: Is this a TODO for now or a later improvement?

**Review by leahwicz — APPROVED** [review]
LGTM

**Comment by lostmygithubaccount** [comment]
@leahwicz @emmyoop looks like it needs approval from someone with write access 😄

**Comment by emmyoop** [comment]
@lostmygithubaccount I think you actually need to resolve the conversation.  That's what's blocking merge.

Not sure why that setting is different on this repo!

**Comment by lostmygithubaccount** [comment]
@emmyoop ohhhh yep, missed that comment! thanks

**Comment by lostmygithubaccount** [comment]
@emmyoop ohhhh yep, missed that comment! thanks

**Pull request merged**
